src/Http/Cookie/Cookie.php

```diff
@@ -251,7 +271,7 @@ public function getName()
      */
     protected function validateName($name)
     {
-        if (preg_match("/[=,; \t\r\n\013\014]/", $name)) {
+        if (preg_match("/[=,;\t\r\n\013\014]/", $name)) {
```

Comment:
If I interpreted that correctly:

> cookie-name       = token
> token             = <token, defined in [RFC2616], Section 2.2>

https://tools.ietf.org/html/rfc6265#section-4.1.1

So we could add:
``@link https://tools.ietf.org/html/rfc2616#section-2.2 Rules for naming cookies.``